spec/models/calculation_result_spec.rb

```diff
@@ -4,4 +4,25 @@
   it "raises an error if there's an unknown result" do
     expect { described_class.new({}).decision }.to raise_error("Unhandled CFE result: ")
   end
+
+  it "works out disregard from net equity minus assessed equity" do
+    data = {
+      assessment: {
+        capital: {
+          capital_items: {
+            properties: {
+              main_home: {
+                value: 20.0,
+                outstanding_mortgage: 9.0,
+                net_equity: "11.0",
+                assessed_equity: "1.0",
```

Comment:
Small niggle - these 2 fields (should now) be decimals not strings back from CFE, after we fixed the string 'defect'

Reply:
Ah yep, I got tripped up by an out-of-date VCR.